ndcube/ndcube_sequence.py

```diff
@@ -6,10 +6,15 @@
 import numpy as np
 
 from ndcube import utils
-from ndcube.mixins.sequence_plotting import NDCubeSequencePlotMixin
 
 __all__ = ['NDCubeSequence']
 
+PLOTTING_NOT_SUPPORTED_ERROR = "NDCubeSequence plotting is no longer supported. " + \
```

Comment:
I would rewrite this to be a single triple quote string?

Reply:
Good idea!